README.md

```diff
@@ -0,0 +1,87 @@
+![Aptabase](https://aptabase.com/og.png)
+
+# Unreal Engine SDK for Aptabase
+
+Instrument your Unreal Engine project with Aptabase, an Open Source, Privacy-First and Simple Analytics for Mobile, Desktop and Web Apps.
+
+## Install
+
+Install the SDK via the [Unreal Marketplace](TODO) or clone this repository inside your `Plugins` folder and compile it (requires C++ based project).
```

Comment:
Is it only the copyright notice that we're missing to publish this to the unreal marketplace?

Reply:
Well, as far as the codebase goes, yes.

To launch the following steps are required:
1. Create an epic seller account - I can take care of this
2. Take the tax interview with your company's info - this will have to be completed by you
3. Product copywriting - I can take care of this
4. Product presentation design - I can take care of this
5. Submission process - I can take care of this

Also, one important note, when submitting the code to Epic, they will distribute it with a custom EULA. People can still come to GitHub and grab it under the MIT license, but people downloading the marketplace are subject to [Marketplace EULA](https://www.unrealengine.com/en-US/marketplace-distribution-agreement)

Reply:
Ok, sounds like a bit of work to do it. So maybe lets gather some interest first, I created this issue ( https://github.com/aptabase/aptabase-unreal/issues/2 ) and linked it on the readme